python_modules/dagster/dagster/_grpc/types.py

```diff
@@ -428,7 +423,6 @@ def __new__(
         partition_name: str,
         job_name: Optional[str] = None,
         instance_ref: Optional[InstanceRef] = None,
-        selected_asset_keys: Optional[AbstractSet[AssetKey]] = None,
```

Comment:
confirming understanding: the fact that this field was always optional and initialized to None is why this is safe to do without breaking older versions that might have been expecting it to be set otherwise

Reply:
exactly